examples/Bunch/include/simulation_defines/param/radiationConfig.param

```diff
@@ -88,6 +88,7 @@ namespace radiationNyquist
    */
   namespace radFormFactor_CIC_3D { }
   namespace radFormFactor_CIC_1Dy { }
+  namespace radFormFactor_Gauss_spherical { }
```

Comment:
New `radFormFactor_Gauss_spherical` function is not mentioned in the comment above.